src/development/LocalTest/Controllers/HomeController.cs

```diff
@@ -197,6 +194,77 @@ public async Task<ActionResult> GetTestOrgToken(string id, [FromQuery] string or
             return await Task.FromResult(Ok(token));
         }
 
+        /// <summary>
+        ///  See src\development\loadbalancer\nginx.conf
+        /// </summary>
+        public static readonly string FRONTEND_URL_COOKIE_NAME = "frontendVersion";
+
+        [HttpGet]
+        public async Task<ActionResult> FrontendVersion([FromServices]HttpClient client)
+        {
+            var versionFromCookie = HttpContext.Request.Cookies[FRONTEND_URL_COOKIE_NAME];
+
+
+
+            var frontendVersion = new FrontendVersion()
+            {
+                Version = versionFromCookie,
+                Versions = new List<SelectListItem>()
+                {
+                    new ()
+                    {
+                        Text = "Standard CDN",
```

Comment:
```suggestion
                        Text = "Don't replace version",
```

Reply:
How about 'keep as is'? Another problem, of course, is if another URL than the latest version 3 is hard-coded in the app.. It's just a matter of time until we release version 4.

Reply:
`Keep as is` is better 👍

When we release version 4 we need another nginx `sub_filter` line that updates scripts with version 4 and a new copy of the perl script that has v4 as default. Another issue is with apps that already pins their version, witch also breaks my current replacement.

To actually fix all versions, we'd need something better than `sub_filter` that accepts a regex as the search pattern, so we can match on all versions. I'm been thinking about abandoning nginx all together and replace with controllers in Localtest that does the same proxying. That would make more complex behavouiour much easier to implement (and also remove the need for docker for many users),

Reply:
Right. I skimmed the docs now, and it doesn't look like it can be done using the nginx perl module that easily. I've done some advanced in-flight replacements with OpenResty and the nginx lua module before, but that's likely overkill here - and it's not like this thing is actually a high traffic load balancer, despite the naming. I'm no C# or dotnet expert, but if it's easy to do there, that might be much better. :+1:

Reply:
It wasn't as easy as I thought to create a proxy (`HttpClient` has lots of helpful? defaults), but I have a draft that works without docker at https://github.com/Altinn/altinn-studio/compare/master...ivarne:localtest-proxy.

Reply:
Nice! I skimmed the code, and it looks good to my untrained eye. Not that much longer than the nginx config file, even. If you're going forward with the prototype, I guess someone other than me should review it, though. :wink: